### PR TITLE
Moved all components *.js files to corresponding subdirectories, move…

### PR DESCRIPTION
**Pull request by Fwerfad**
…d their stories there. Created index.js file for each component for pretty import.

Fixed imports from components

### CODE REVIEW

**Review by Mixxx-a — COMMENTED** [review]
Please resolve merge conflicts

**Review by StanislavMyakishev — APPROVED** [review]

File: components/FilterBar/FilterBar.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useRecoilState } from 'recoil'
-import InputContainer from './InputContainer'
-import { formFiltersState } from './Catalog'
+import InputContainer from '../InputContainer/InputContainer'
```
Comment (StanislavMyakishev) [inline components/FilterBar/FilterBar.js:2]: Components should be imported from the folder root - index.js so there won't be any excess in path

File: pages/index.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import Header from '../components/Header'
-import Search from '../components/Search'
+import Header from '../components/Header/Header'
```
Comment (StanislavMyakishev) [inline pages/index.js]: Check this file as well

File: pages/wine/[id].js [hunk 3]
```diff
@@ -2,15 +2,15 @@ import React from 'react'
...
 import { selectorFamily, useRecoilValueLoadable } from 'recoil'
 
-import WinePosition from '../../components/WinePosition'
-import ItemDescription from '../../components/ItemDescriptionCard'
-import ReviewBox from '../../components/ReviewBox'
-import Header from '../../components/Header'
-import Search from '../../components/Search'
-import Wrapper from '../../components/Wrapper'
-import SameWines from '../../components/SameWines'
+import WinePosition from '../../components/WinePosition/WinePosition'
```
Comment (StanislavMyakishev) [inline pages/wine/[id].js:5]: And last but not least :)

**Review by StanislavMyakishev — CHANGES_REQUESTED** [review]
Should use components import via index.js files so

**Pull request closed without merging**